README.md

```diff
@@ -1,13 +1,81 @@
 # GridGopher
 
-A placeholder repository for Allegheny College automated grade transfer tool
+![License](https://img.shields.io/badge/license-MIT-blue?style=flat)
+![BuiltWith](https://img.shields.io/badge/Built%20With-Python-blue?style=flat&logo=python&logoColor=yellow)
+![Actions Status](https://github.com/noorbuchi/GridGopher/workflows/Lint%20and%20Test/badge.svg)
+[![codecov](https://codecov.io/gh/noorbuchi/GridGopher/branch/main/graph/badge.svg?token=02353FAN4W)](https://codecov.io/gh/noorbuchi/GridGopher)
+![stars](https://img.shields.io/github/stars/noorbuchi/GridGopher.svg)
 
-## Important Notes
+![GridGopherLogo](images/GopherLogo.png)
 
-- Values in the `.env` file must be surrounded by double quotation marks `"`
-  otherwise, newline character `\n` will cause issues.
+GridGopher is a plugin friendly tool that allows users to connect collected data
+from Google Sheets and GitHub issue trackers and pull requests. The tool
```

Comment:
I noticed that the terms 'issue trackers' and 'pull requests' were used multiple times in the readme but have different cases throughout. Consider capitalizing the terms to keep things uniform.